features/integration/compile.feature

```diff
@@ -27,5 +27,5 @@ Feature: Compile
     When I compile a teal program "<teal>" with mapping enabled
     Then the resulting source map is the same as the json "<sourcemap>"
     Examples:
-      | teal                | sourcemap                                   |
-      | programs/quine.teal | v2algodclient_responsejsons/sourcemap2.json |
\ No newline at end of file
+      | teal                | sourcemap                                  |
```

Comment:
Not sure why this is failing again. Might be a flaky test that depends on the order run in comparison with other features/steps.

Reply:
just a dumb question, maybe I don't know what's behind the curtain but, `sourcemap.json` and `sourcemap2.json` are not the same, why replacing to a different `sourcemap.json` for a same teal file solves some problem?

Reply:
Originally, before I worked on sandboxization, they were the same. Then it started failing, and I was getting different results for the integration test. Looking at the results, I [crafted a new `sourcemap2.json`](https://github.com/algorand/algorand-sdk-testing/pull/220/commits/63f0947116e87a6afa299c9f36698c8170f90db7) and got the tests to pass. Now, the bug seems to have gone away. Maybe it has to do with different behavior against `go-algorand`'s `master` v. `feature/avm-box`. I'm not sure and that's what I'm going to investigate in #226.

Reply:
yea, that matches with my hunch too, that something inside go-algorand produces different sourcemap behavior, and thus the old `sourcemap.json` fail to match expected results.

Reply:
PR to fix this: https://github.com/algorand/algorand-sdk-testing/pull/227

I'm ready to approve this PR after the above one is merged